Dockerfile

```
@@ -0,0 +1,44 @@
+# 4. install oh-my-zsh for virgo & set bureau theme
+ARG OH_MY_ZSH_REF=master
+ARG OH_MY_ZSH_INSTALL_SHA256=REPLACE_WITH_PINNED_INSTALL_SH_SHA256
+USER virgo
+RUN curl -fsSL "https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/${OH_MY_ZSH_REF}/tools/install.sh" -o /tmp/install-oh-my-zsh.sh && \
+    printf '%s  %s\n' "${OH_MY_ZSH_INSTALL_SHA256}" "/tmp/install-oh-my-zsh.sh" | sha256sum -c - && \
+    sh /tmp/install-oh-my-zsh.sh "" --unattended && \
+    rm -f /tmp/install-oh-my-zsh.sh && \
+    sed -i 's/ZSH_THEME="robbyrussell"/ZSH_THEME="bureau"/' ~/.zshrc
+
+# 5. finalize environment
+USER root
```

Hadolint (2.14.0)

[warning] Last USER should not be root

(DL3002)